wapitiCore/attack/mod_wapp.py

```diff
@@ -90,15 +91,27 @@ def attack(self):
             self.log_blue("---")
 
         for application_name in sorted(detected_applications, key=lambda x: x.lower()):
+            
+            versions = detected_applications[application_name]["versions"]
+            categories = detected_applications[application_name]["categories"]
+
             self.log_blue(
                 MSG_TECHNO_VERSIONED,
                 application_name,
-                detected_applications[application_name]["versions"]
+                versions
             )
             self.add_addition(
```

Comment:
This function call should be made inside an `else` block bellow otherwise the web server may appear in both categories.

Reply:
@fwininger do we need web server detection both in vulnerability and additional parts of the report ?